RecoEgamma/EgammaElectronProducers/python/ecalDrivenElectronSeedsParameters_cff.py

```diff
@@ -34,7 +34,11 @@
     # H/E towers
     hcalTowers = cms.InputTag("towerMaker"),
     hOverEPtMin = cms.double(0.),
-    
+
+    applySigmaIEtaIEtaCut = cms.bool(True),
+    # sigma_ietaieta
+    maxSigmaIEtaIEtaBarrel = cms.double(0.5),
+    maxSigmaIEtaIEtaEndcaps = cms.double(0.5),    
```

Comment:
Hi Lindsey,



On Tue, Jul 14, 2015 at 3:08 PM, Lindsey Gray <notifications@github.com>
wrote:

> In
> RecoEgamma/EgammaElectronProducers/python/ecalDrivenElectronSeedsParameters_cff.py
> <https://github.com/cms-sw/cmssw/pull/10194#discussion_r34565363>:
>
> > @@ -34,7 +34,11 @@
> >      # H/E towers
> >      hcalTowers = cms.InputTag("towerMaker"),
> >      hOverEPtMin = cms.double(0.),
> > -
> > +
> > +    applySigmaIEtaIEtaCut = cms.bool(True),
> > +    # sigma_ietaieta
> > +    maxSigmaIEtaIEtaBarrel = cms.double(0.5),
> > +    maxSigmaIEtaIEtaEndcaps = cms.double(0.5),
>
> If so, it should be turned off here and this module should be customized
> such that it is turned on for HI reconstruction.
>

Do you mean I have to customize this module for HI reconstruction?

Thanks,
Hien.

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/10194/files#r34565363>.
>